README.md

```diff
@@ -43,6 +43,17 @@ $ ./circleci/catapult-publish [CATAPULT_URL] [CATAPULT_USER] [CATAPULT_PASS] [AP
 
 If you need to publish multiple applications, run this command once for each.
 
+### Dapple
```

Comment:
Let's move this in a seperate section which is Clever only. Okay with leaking the name and URIs but don't think we can make dapple public yet. Let's make that explicit